### PR TITLE
🗃️ 11 - add participant, question and response schemas

### PR DESCRIPTION
**Pull request by anncatton**
Renames `User` model to `Participant`. Deletes previous init migration that created the `User` table
Adds `ConsentQuestion` and `ParticipantResponse` models + migrations.
- the `on delete` and `on update` relationship for a participant response with participants and consent questions is currently set to *not* perform any cascading changes. This may need to be revisited depending on requirements.

### CODE REVIEW

File: apps/api/prisma/migrations/20230525183353_init/migration.sql
Comment (mistryrn) [inline apps/api/prisma/migrations/20230525183353_init/migration.sql:1]: #rip

File: apps/api/prisma/schema.prisma [hunk 1]
```diff
@@ -7,8 +7,36 @@ generator client {
   provider = "prisma-client-js"
 }
 
-model User {
+model Participant {
```
Comment (mistryrn) [inline apps/api/prisma/schema.prisma:10]: 🥲

**Review by mistryrn — APPROVED** [review]
Good stuff! 👍

File: apps/api/prisma/migrations/20230525183353_init/migration.sql
Comment (anncatton) [reply apps/api/prisma/migrations/20230525183353_init/migration.sql:1]: lol...we hardly knew ye

**Pull request merged**